### PR TITLE
DE36955 - fix rubric assessment table cells in firebox

### PR DESCRIPTION
**Pull request by xiangqiyan**

### CODE REVIEW

File: d2l-rubric-criteria-group.js [hunk 1]
```diff
@@ -345,6 +348,16 @@ Polymer({
 		'_updateFeedbackDisplay(_criteriaEntities, assessmentResult)'
 	],
 
+	_rowHeaderDomChange: function() {
+		// set styling to have the criteria-row-header-container be the same height as the table cell in firefox
+		if (navigator.userAgent.toLowerCase().indexOf('firefox') > -1) {
```
Comment (xiangqiyan) [inline d2l-rubric-criteria-group.js]: This issue happened after I made the change to have the add feedback button show at the bottom of the cell. I couldn't find a way to style the table that works for both chrome and firefox so i did this as a workaround.

Comment (mpharoah-d2l) [reply d2l-rubric-criteria-group.js]: What about Edge?

Comment (xiangqiyan) [reply d2l-rubric-criteria-group.js]: edge behaves the same as chrome

Comment (mpharoah-d2l) [reply d2l-rubric-criteria-group.js]: Doing a case sensitive check for `Firefox/` is safer (https://developer.mozilla.org/en-US/docs/Web/HTTP/Browser_detection_using_the_user_agent#Browser_Name)

**Pull request merged**